### DISCUSSION TITLE
WebGL warning on LibreWolf

### DISCUSSION THREAD

**Discussion by ashkan117** [pmndrs/zustand#2588] [Bug report]
When opening [the demo site](https://zustand-demo.pmnd.rs/) with [LibreWolf](https://librewolf.net/) it was all black so at first I thought it was not working. However I switched a Chromium browser things worked fine. 

```bash
THREE.WebGLRenderer: A WebGL context could not be created. Reason:  WebGL is currently disabled.
THREE.WebGLRenderer: Error creating WebGL context.
```

Just wanted to bring this to your attention so that maybe a fallback is rendered in cases like these?

**Comment by dai-shi** [comment]
Not quite related, but @sunil-sharma-999 is working on some fixes in #2585.
